e2e/content-services/search/components/search-text.e2e.ts

```diff
@@ -0,0 +1,122 @@
+import path = require('path');
+
+describe('Search component - Text widget', () => {
+
+    const searchCategoriesPage = new SearchCategoriesPage();
+    const configEditorPage = new ConfigEditorPage();
+    const navigationBarPage = new NavigationBarPage();
+    const searchFiltersPage = new SearchFiltersPage();
+
+    let loginPage = new LoginPage();
+    let searchDialog = new SearchDialog();
+    let searchResultPage = new SearchResultsPage();
+
+    let acsUser = new AcsUserModel();
+    let newFolderModel = new FolderModel({ 'name': 'newFolder' , 'description': 'newDescription' });
+
+    beforeAll(async (done) => {
+
+        this.alfrescoJsApi = new AlfrescoApi({
+            provider: 'ECM',
+            hostEcm: TestConfig.adf.url
+        });
+
+        await this.alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+
+        await this.alfrescoJsApi.core.peopleApi.addPerson(acsUser);
+
+        await this.alfrescoJsApi.login(acsUser.id, acsUser.password);
+
+        await this.alfrescoJsApi.nodes.addNode('-my-', {
+            'name': newFolderModel.name,
+            'description': newFolderModel.description,
+            'nodeType': 'cm:folder'
+        }, {}, {});
+
+        loginPage.loginToContentServicesUsingUserModel(acsUser);
+
+        done();
+    });
+
+    it('[C289329] Placeholder should be displayed in the widget when the input string is empty', () => {
+        browser.get(TestConfig.adf.url + '/search;q=*');
+        searchResultPage.tableIsLoaded();
+
+        searchFiltersPage.checkNameFilterIsDisplayed();
+        expect(searchFiltersPage.textFiltersPage().getNamePlaceholder()).toEqual('Enter the name');
+    });
+
+    it('[C289330] Should be able to change the Field setting', () => {
+        browser.get(TestConfig.adf.url + '/search;q=*');
+        searchResultPage.tableIsLoaded();
+
+        searchFiltersPage.checkCheckListFilterIsDisplayed();
+        searchFiltersPage.clickCheckListFilter();
+        searchFiltersPage.checkListFiltersPage().clickCheckListOption('Folder');
+
+        searchFiltersPage.checkNameFilterIsDisplayed();
+        searchFiltersPage.textFiltersPage().searchByName(newFolderModel.name);
+        searchResultPage.checkContentIsDisplayed(newFolderModel.name);
+
+        searchFiltersPage.textFiltersPage().searchByName(newFolderModel.description);
+        searchResultPage.checkContentIsNotDisplayed(newFolderModel.name);
+
+        var json = JSON.parse(require('fs').readFileSync(path.join(TestConfig.main.rootPath, '/content-services/search/search.config.json'), 'utf8'));
+
+        json.categories[0].component.settings.field = 'cm:description';
+
+        navigationBarPage.clickConfigEditorButton();
+        configEditorPage.clickSearchConfiguration();
+        configEditorPage.clickClearButton();
+
+        configEditorPage.enterConfiguration(JSON.stringify(json));
+        // configEditorPage.enterConfiguration("{  edcefefef wefweffwfwaef wefwefwef}");
```

Comment:
I think this comment here can be removed.

Reply:
I'll remove it, thanks